editor/editor_node.cpp

```diff
@@ -3045,6 +3045,19 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			}
 
 			if (!p_confirmed) {
+				if (project_run_bar->is_playing()) {
+					if (p_option == PROJECT_RELOAD_CURRENT_PROJECT) {
+						confirmation->set_text(TTR("Stop running project before reloading the current project?"));
+						confirmation->set_ok_button_text(TTR("Stop & Reload"));
+					} else {
+						confirmation->set_text(TTR("Stop running project before exiting the editor?"));
+						confirmation->set_ok_button_text(TTR("Stop & Quit"));
```

Comment:
This should use TTRC.

Reply:
@KoBeWi Sounds good, but what's happening here specifically? I notice that both TTR and TTRC are used in the file in different places 🤔

Reply:
TTR marks a string for translation and translates string in place.
TTRC only marks it, but does nothing otherwise.
("marked" strings are extracted by an automated script that does regex on the codebase)

Godot has auto-translation system, which wasn't active in the editor in the past, but it was changed recently. Thus TTR strings are subject to translation twice (by TTR function and then auto-translation). Using TTRC allows for auto-translation only, which makes it easier to switch languages (see #102562).